github/repos_rules.go

```diff
@@ -134,7 +134,7 @@ func (r *RepositoryRule) UnmarshalJSON(data []byte) error {
 	r.Type = RepositoryRule.Type
 
 	switch RepositoryRule.Type {
-	case "creation", "deletion", "required_linear_history", "required_signatures", "non_fast_forward":
+	case "creation", "deletion", "required_linear_history", "required_signatures", "non_fast_forward", "merge_queue":
```

Comment:
I know these weren't sorted before, but would you mind sorting them now, please?